rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
```

Comment:
- [ ] The constructor of Rental should be modified so Book and Person are set in it to break the `many-to-many ` relationship, something like so: 
```
`def initialize(date, book, person)`

```